lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml

```diff
@@ -23,12 +22,41 @@ firefox: BROWSER=firefox
 chrome: BROWSER=chrome
 ie: BROWSER=ie
 safari: BROWSER=safari
```

Comment:
Only leave firefox and chrome in this template as discussed

Reply:
Woops, forgot to include that bit of code. Already have it locally.